Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Surfaces/StandardSurface.azsli

```diff
@@ -78,6 +78,12 @@ class Surface
 
     //! Sets albedo, base color, specularF0, and metallic properties using metallic workflow
     void SetAlbedoAndSpecularF0(real3 baseColor, real specularF0Factor, real metallic);
+
+    real3 GetDiffuseNormal() { return normal; }
+    real3 GetSpecularNormal() { return normal; }
+    real3 GetDefaultNormal() { return normal; }
```

Comment:
I'm not super clear on the use of DefaultNormal, some comments might be helpful here.

Reply:
+1 Default in what context? Is there a more specific name that can be used here?